.github/workflows/ci-pr-validation.yml

```
@@ -108,8 +108,37 @@ jobs:
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd #v6.0.2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python
+        uses: actions/setup-python@42375524ee3024772b18e3fcb589fdd9aeb36f5d #v6.0.0
+        with:
+          python-version: '3.12'
+
+      - name: Write PR body to file
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)